recipes/ffmpeg/bld.bat

```diff
@@ -0,0 +1,3 @@
+:: http://ffmpeg.zeranoe.com/blog/?p=300
+
+bash "%RECIPE_DIR%\mingw-w64-build-3.6.0.sh"
```

Comment:
Binaries of what? MinGW or ffmpeg? I have no problem *using* mingw to compile things - I'm just keen to package the useful things that need packaging (e.g. https://github.com/conda-forge/msinttypes-feedstock)